### PR TITLE
Fix vich_config - property name, UserManager::hasRole(), hardcoded template path to prototype, UserLoad

### PR DESCRIPTION
**Pull request by bubnovKelnik**
There was an error in Hackzilla\Bundle\TicketBundle\Entity\TicketMessageWithAttachment.xml, wrong field name value (file).
Fixed value - "attachmentFile"

### CODE REVIEW

**Comment by bubnovKelnik** [comment]
Are going to fix some to pass all checks :) Will come back soon!

File: Manager/UserManager.php [hunk 1]
```diff
@@ -61,7 +82,7 @@ public function getUserById($userId)
      */
     public function hasRole(UserInterface $user, $role)
     {
-        return in_array(strtoupper($role), $user->getRoles(), true);
+        return $this->authorizationChecker->isGranted($role);
```
Comment (hackzilla) [inline Manager/UserManager.php]: Resources/views/Ticket/show.html.twig
```
{% if message.userObject|isTicketAdmin('ROLE_TICKET_ADMIN') %}<span
   class="label label-danger">{{ 'LABEL_ADMIN'|trans }}</span> {% endif %}
```

This checks the user of the message to see if they are admin.  I.e. not always the logged in user.  

screen shot 2016-10-11 at 08 26 56

I can't think of any way around this, other than storing this information in the db.

Comment (bubnovKelnik) [inline Manager/UserManager.php]: Hm.. What is it label for?

Comment (bubnovKelnik) [inline Manager/UserManager.php]: Why should admin post a ticket?

Comment (bubnovKelnik) [inline Manager/UserManager.php]: If this is some kind of support - the author of the ticket is user with his question.
There are only 2 types of users able to write new messages in the ticket - the author and support member.
If you want to mark messages from support with "Admin" label - you just check if message.userObject != ticket.userCreatedObject

Comment (bubnovKelnik) [inline Manager/UserManager.php]: I could do this PR in show.html.twig

Comment (bubnovKelnik) [inline Manager/UserManager.php]: Have sent you the PR

Comment (bubnovKelnik) [inline Manager/UserManager.php]: After this PR there is no need in UserExtension anymore - may be remove it?

Comment (hackzilla) [inline Manager/UserManager.php]: Yeah, that makes sense.

**Comment by hackzilla** [comment]
Thank you for your hard work!

**Comment by bubnovKelnik** [comment]
:^)

**Pull request merged**